src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -430,4 +433,25 @@ bool ShouldScanDependencies(Assembly assembly)
         // And other windows azure stuff
         "Microsoft.WindowsAzure"
     };
+
+    static AssemblyScanner()
```

Comment:
As an alternative this could also be moved into a lazy to pay the price only once when GetScannableAssembly is called.